src/cli/trn_cli_common.c

```diff
@@ -442,6 +442,87 @@ int trn_cli_parse_ep_key(const cJSON *jsonobj,
 	return 0;
 }
 
+int trn_cli_parse_packet_metadata(const cJSON *jsonobj, struct rpc_trn_packet_metadata_t *packet_metadata)
+{
+	cJSON *tunnel_id = cJSON_GetObjectItem(jsonobj, "tunnel_id");
+	cJSON *ip = cJSON_GetObjectItem(jsonobj, "ip");
+	cJSON *pod_label_value = cJSON_GetObjectItem(jsonobj, "pod_label_value");
+	cJSON *namespace_label_value = cJSON_GetObjectItem(jsonobj, "namespace_label_value");
+
+	if (tunnel_id == NULL) {
+		packet_metadata->tunid = 0;
```

Comment:
Is this a valid tunnel id? Of should we return error?

Reply:
Yes it's valid. endpoint code does the same thing.

Reply:
The 0 case is for updating the endpoints about hosts/droplets.
Unless network policy also affects hosts/droplets 0 should be invalid.